### DISCUSSION TITLE
cloud.walletconnect.com - Cannot create project

### DISCUSSION THREAD

**Discussion by adrienthiery** [WalletConnect/walletconnect-monorepo#3556] [Developer Support] [answered]
Hi there,
I wanted to create a new project and I'm stuck with a `new row violates row-level security policy for table "project_quotas"` error just after signup.
```
instrument.js:137     POST https://awyuaxlpezyhfkpznrox.supabase.co/rest/v1/projectscolumns=%22name%22%2C%22creator%22&select=* 403
```

Return payload: 
```
code: "42501"
details: null
hint: null,
message: "new row violates row-level security policy for table \"project_quotas\""
```

**Comment by 3700rpm** [comment]
SAME zzz im in rush

**Comment by Sayeh-1337** [comment]
Same Here

**Comment by freewilliam** [comment]
➕ 1️⃣

**Comment by Fuga14** [comment]
Same problem, even tried to create new acc, didn't help

**Comment by toniocodo** [comment]
Same

**Comment by lyzetam** [comment]
Same problem: new row violates row-level security policy for table "project_quotas"

**Comment by elliotBraem** [comment]
Me too, would like ASAP

**Comment by r4kh4t** [comment]
Same here, also cannot receive an email confirmation letter

**Comment by huazhuangnan** [comment]
me too

**Comment by ghost** [comment]
me too

**Comment by vebester** [comment]
Same problem: new row violates row-level security policy for table "project_quotas"

**Comment by glitch-txs** [comment]
Hi everyone, we're aware of this issue and we're currently working on a fix, thanks for reporting 🙏

**Comment by glitch-txs** [comment, marked as answer]
Hi everyone, should be fixed now

**Reply by vebester** [reply]
works

**Reply by adrienthiery** [reply]
Thanks @glitch-txs !

**Reply by huazhuangnan** [reply]
Thanks @glitch-txs
